### DISCUSSION TITLE
Settings Window Width

### DISCUSSION THREAD

**Discussion by hugok79** [SubtitleEdit/subtitleedit#6356] [General]
Hi @nikolajolsson @niksedk
It is possible to increase the width of the Settings window so that all the translation text appears without having to shorten the translation?

![Captura de ecrã 2022-10-25 190237](https://user-images.githubusercontent.com/16929048/197849441-771c0a60-5881-4850-88a0-06a1ad333a53.png)

**Comment by nikse** [comment]
@'d the wrong person, i'm not involved in this project 🤦‍♂️

**Reply by hugok79** [reply]
Sorry, my mistake!

**Comment by hugok79** [comment]
![Captura de ecrã 2022-10-26 121546](https://user-images.githubusercontent.com/16929048/198013941-3c200ff4-3261-4704-82e9-d48231f3c8e9.png)
![Captura de ecrã 2022-10-26 121459](https://user-images.githubusercontent.com/16929048/198013944-34120685-cedb-4f51-a485-7c554504ae23.png)

**Reply by hugok79** [reply]
Some more examples where the entire translation does not appear in the text box...
